services/conference/src/scripts/components/utils/KeyHandler.ts

```diff
@@ -0,0 +1,90 @@
+import {useEffect, useRef} from 'react'
+
+export class KeyHandlerReact<ET extends Element>{

```

Comment:
It seems both KeyHandlerReact and KeyHandlerPlain are not used.

Reply:
Sorry I've used these function later. But included in this version.

Reply:
Thank you for finding bug of Fab icons. I'll fix them soon.